src/OAuth2Form.tsx

```diff
@@ -180,7 +180,10 @@ export default class OAuth2Form
             redirectUri: this.getURLParam('redirect_uri')!,
             nonce: this.getURLParam('nonce')
         }
-        return authRequest
+        if (authRequest.claims !== undefined) {
```

Comment:
Could you change this to any falsy value (undefined, '' (empty string), null, NaN, 0, or false)?  Then it would be:
if (authRequest.claims) {
...
}